### PR TITLE
Bumped BT SDK version

### PR DESCRIPTION
**Pull request by stechiu**
### Summary of changes

 - Bumped with latest BT SDK version 

 ### Checklist

 - [x] Added a changelog entry

- @stechiu

### CODE REVIEW

**Review by agedd — APPROVED** [review]
lgtm 🚀

File: BraintreeDropIn.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "BraintreeDropIn"
-  s.version          = "9.12.2"
+  s.version          = "9.13.0"
```
Comment (jaxdesmarais) [inline BraintreeDropIn.podspec:3]: We don't want to manually change the version, this is updated automatically via CI when we do a release

File: BraintreeDropIn.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 2]
```diff
@@ -5,17 +5,17 @@
...
     },
     {
       "identity" : "inappsettingskit",
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/futuretap/InAppSettingsKit.git",
       "state" : {
-        "revision" : "71e03ebad8462618f96c575c86d2e9220f880676",
-        "version" : "3.3.2"
+        "revision" : "5b6a33cd5d227d71f79bf8e43be2049d981c6aad",
+        "version" : "3.7.1"
```
Comment (jaxdesmarais) [inline BraintreeDropIn.xcworkspace/xcshareddata/swiftpm/Package.resolved:18]: Lets revert this, we can make this change in a separate PR if needed

File: Demo/Application/Supporting Files/Braintree-Demo-Info.plist [hunk 3]
```diff
@@ -29,7 +29,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>9.12.2</string>
+	<string>9.13.0</string>
```
[the remark this answers is not in the document]
Comment (stechiu) [reply Demo/Application/Supporting Files/Braintree-Demo-Info.plist:32]: Didn't realize CI will update it automatically. Thanks!

File: BraintreeDropIn.xcworkspace/xcshareddata/swiftpm/Package.resolved (hunk 2 above)
Comment (jaxdesmarais) [inline BraintreeDropIn.xcworkspace/xcshareddata/swiftpm/Package.resolved:18]:
```suggestion
        "revision" : "71e03ebad8462618f96c575c86d2e9220f880676",
        "version" : "3.3.2"
```

File: Demo/Application/Supporting Files/Braintree-Demo-Info.plist (hunk 3 above)
[the remark this answers is not in the document]
Comment (jaxdesmarais) [reply Demo/Application/Supporting Files/Braintree-Demo-Info.plist]: If you're curious this is where that logic lives: https://github.com/braintree/braintree-ios-drop-in/blob/15b674d700e120bb66f9312b2866bfed472d938d/.github/workflows/release.yml#L32-L42

**Review by jaxdesmarais — APPROVED** [review]
